### ISSUE TITLE
_was only populated on change

### ISSUE THREAD

**Issue by maxsalven** [lwe/simple_enum#109]
The column_was values are nil if the column value did not change. This does not seem to match the default Rails behavior? e.g. id_was always returns the id in Rails, despite this value not changing.

**Comment by lwe** [comment]
Hi @maxsalven, can you provide some information about which version of ruby, rails and simple_enum you are using and can you provide me the output of the following calls:

```ruby
# The Rails column / attribute - please of course replace didum with your column name :)
model.didum_cd
model.didum_cd_was

# The simple_enum generated column
model.didum
model.didum_was
```

and how you are using the `as_enum` call, e.g. a line like

```ruby
as_enum :didum, .....
```

**Comment by maxsalven** [comment]
````
ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14]
Rails 4.2.3
simple_enum (2.1.1)
```

````ruby
# result.rb
as_enum :status, [:completed, :cap, :dnf, :dns], map: :string, source: :status
````

````
max$ rails c
Loading development environment (Rails 4.2.3)
[1] pry(main)> r = Result.first
...
[2] pry(main)> r.id_was
=> 1
[3] pry(main)> r.status_was
=> nil
[4] pry(main)> r.status = 'cap'
=> "cap"
[5] pry(main)> r.status_was
=> :completed
[6] pry(main)> r.status = 'completed'
=> "completed"
[7] pry(main)> r.status_was
=> nil
````

**Comment by lwe** [comment]
Thanks. It looks like the culprit is that source == enum name...

**Comment by lwe** [comment]
Does this fix work for you #110
